mantis-server/mantis-server-agent/src/main/java/io/mantisrx/server/agent/BlobStore.java

```diff
@@ -118,7 +118,7 @@ public File get(URI blobUrl) throws IOException {
             {
                 return blobStore.get(blobUrl);
```

Comment:
Shouldn't the blobstore only be throwing IOException?

Reply:
tested this one and the actual error thrown on access deny is runtime exception.